resources/definitions/fdmprinter.def.json

```diff
@@ -4412,6 +4412,21 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "retraction_combing_avoid_distance":
+                {
+                    "label": "Distance to Walls",
```

Comment:
Not a huge fan of this name. I'd probably call it something like "Distance to walls when combing", as "Distance to walls" is a bit too generic and can be confusing. 

Perhaps we could also use "Combing avoid distance" or something like that?

Reply:
I totally agree with changing it. We already have `Travel Avoid Distance` though, which is the avoidance distance around the models. Maybe `Inside Travel Avoid Distance` ?